src/platform/forms/save-in-progress/SaveInProgressIntro.jsx

```diff
@@ -60,38 +60,52 @@ class SaveInProgressIntro extends React.Component {
 
         if (!isExpired) {
           const lastSavedDateTime = savedAt.format('M/D/YYYY [at] h:mm a');
-
-          alert = (
-            <div>
-              <div className="usa-alert usa-alert-info background-color-only schemaform-sip-alert">
-                <div className="schemaform-sip-alert-title">
-                  <strong>Your {appType} is in progress</strong>
```

Comment:
My understanding of the problem is that this `<strong>` would be better off as an `h3` in some situations and an `h2` in others. Is that correct?

Reply:
You are correct.

Reply:
I left some [feedback on the other PR](https://github.com/department-of-veterans-affairs/vets-website/pull/17282#discussion_r636359877) linked in the slack thread that I think is applicable here as well. That may result in fewer changes and a solution that is more general purpose. What do you think?

Reply:
Yes, I am all for using the other PR as the foundation for this fix as well. I like the Functional component solution you proposed I wasn't aware you could do stuff like that in React. it's pretty nifty. 

I am waiting to go further with this until that PR gets resolved.